src/cairo_types/mod.rs

```diff
@@ -1,6 +1,7 @@
 pub(crate) mod bigint;
 pub(crate) mod builtins;
 pub(crate) mod dict_access;
+pub(crate) mod new_syscalls;
```

Comment:
"new" isn't very clear to me. suggestions:

* `cairo1_syscalls` (may not be strictly correct)
* `current_syscalls`
* a brief explanation inside `new_syscalls.rs` about new vs. deprecated

Reply:
There is a corresponding new_syscalls.cairo file.